_posts/tommytester/2022-09-22-first-post-tommytestertwo.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+author: tommytester
+title: "Tommytester's first post!"
+---
+
+Tommytestertwo: much better than the original.
+
+This stuff down here is _Markdown_, believe it **or not**!
```

Comment:
This is how I'd like you to report any typos you want changed this first time around.